src/ducks/pin/PinAuth.jsx

```diff
@@ -112,9 +112,10 @@ class PinAuth extends React.Component {
     this.logout()
   }
 
-  logout() {
+  async logout() {
     const { client } = this.props
-    client.logout()
+    await client.logout()
+    window.location.reload()
```

Comment:
Why do we need to reload the page after client logout?

Reply:
Since on desktop the page is not wrapped around authentication (authentication being served by the stack), nothing happens otherwise after logout.